proto/lightning.proto

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2013-2017 The btcsuite developers
+// Copyright (C) 2015-2016 The Decred developers
+// Copyright (C) 2015-2022 Lightning Labs and The Lightning Network Developers
+// Copyright (C) 2022 Bottlepay and The Lightning Network Developers
+
+syntax = "proto3";
+
+package proto;
+
+option go_package = "github.com/bottlepay/lndsigner/proto";
+
+/*
+ * Comments in this file will be directly parsed into the API
+ * Documentation as descriptions of the associated method, message, or field.
+ * These descriptions should go right above the definition of the object, and
+ * can be in either block or // comment format.
+ *
+ * An RPC method can be matched to an lncli command by placing a line in the
+ * beginning of the description in exactly the following format:
+ * lncli: `methodname`
+ *
+ * Failure to specify the exact name of the command will cause documentation
+ * generation to fail.
+ *
+ * More information on how exactly the gRPC documentation is generated from
+ * this proto file can be found here:
+ * https://github.com/lightninglabs/lightning-api
+ */
+
+// Lightning is the main RPC server of the daemon.
+service Lightning {
```

Comment:
Pretty ugly api design with these subservers carrying over from standard lnd. Not much we can do about it I guess.

Reply:
Agree for now, maybe we can figure something out to make it less ugly eventually.